cmd/executor/main_test.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/allegro/mesos-executor"
+	"github.com/mesos/mesos-go/api/v1/lib/executor/config"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestExecutorStartsWithoutCheckpointConfig(t *testing.T) {
+	// given
+	conf := executor.Config{
+		Debug: true,
+		MesosConfig: config.Config{
+			FrameworkID: "framework",
+			ExecutorID:  "executor",
+			AgentEndpoint: "localhost",
+		},
+	}
+
+	// when
+	err := startExecutor(conf)
+
+	// then
+	assert.Error(t, err)
+	assert.Equal(t, "failed to re-establish subscription with agent within 1s, aborting", err.Error())
```

Comment:
How do we know that it will take exactly one second to produce error? In addition, why do we make an assertion for an error message? Changing the text is supposed to fail the tests?

Reply:
Done.